ncm-systemd/src/main/pan/components/systemd/schema.pan

```diff
@@ -321,6 +321,57 @@ type ${project.artifactId}_unitfile_config_socket = {
     true;
 };
 
+@documentation{
+the [mount] section
+http://www.freedesktop.org/software/systemd/man/systemd.mount.html
+}
+type ${project.artifactId}_unitfile_config_mount = {
+    include ${project.artifactId}_unitfile_config_systemd_exec
+    include ${project.artifactId}_unitfile_config_systemd_kill
+    'What': string
+    'Where': absolute_file_path
+    'Type' ? string
+    'Options' ? string[]
+    'SloppyOptions' ? boolean
+    'LazyUnmount' ? boolean
+    'ReadWriteOnly' ? boolean
+    'ForceUnmount' ? boolean
+    'DirectoryMode' ? type_octal_mode
+    'TimeoutSec' ? long(0..)
+};
+
+@documentation{
+the [Automount] section
+http://www.freedesktop.org/software/systemd/man/systemd.automount.html
+}
+type ${project.artifactId}_unitfile_config_automount = {
+    'Where': absolute_file_path
+    'DirectoryMode' ? type_octal_mode
+    'TimeoutSec' ? long(0..)
+};
+
+@documentation{
+the [Timer] section
+http://www.freedesktop.org/software/systemd/man/systemd.timer.html
+}
+type ${project.artifactId}_unitfile_config_timer = {
+    'OnActiveSec' ? long(0..)
+    'OnBootSec' ? long(0..)
+    'OnStartupSec' ? long(0..)
+    'OnUnitActiveSec' ? long(0..)
+    'OnUnitInactiveSec' ? long(0..)
+    'OnCalendar' ? string[]
```

Comment:
I'm not sure what to do with this one. It supports a lot of different syntax: https://www.freedesktop.org/software/systemd/man/systemd.time.html#

Reply:
Oh wow.

Either leave it as is, or pick a subset of the supported syntax to validate.

Reply:
OK, leave as string.